<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionAnalysis</class>
 <widget class="QDialog" name="FunctionAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>1491</width>
     <height>841</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>879</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>1021</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>FuncScan: Apply co-expression analysis to predict the functions of the novel transcripts</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>1371</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>290</y>
       <width>701</width>
       <height>300</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial [TMC ]</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Select gene for co-expression and functional enrichment analysis </string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>591</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Human</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mouse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zebrafish</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>input gene id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_id">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="line_pathway">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Arial [TMC ]</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>snakemake/script/pathway/hg38_gene2File.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Gene Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_geneset">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="browse_geneset">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="begin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="example_result">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial [TMC ]</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Example Result</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>661</width>
       <height>300</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial [TMC ]</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>View the results of unannotated marker gene functional enrichment analysis </string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>591</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Arial [TMC ]</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>chose unannoated gene to plot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_lnc">
         <property name="font">
          <font>
           <family>Arial [TMC ]</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="browse_lncRNA">
         <property name="font">
          <font>
           <family>Arial [TMC ]</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial [TMC ]</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
